### PR TITLE
Parameter adjustment

### PR DESCRIPTION
**Pull request by AndyZzzZzzZzz**
- Added tile as a parameter to the raster_embedding_search function.
- Added conditional checks to raster_embedding_search function.

### CODE REVIEW

File: minorminer/utils/raster_embedding.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def generator(raster_breadth):
     return raster_breadth
 
 def raster_embedding_search(S, T, timeout=10, raster_breadth=None,
-                            max_num_emb=float('Inf')):
+                            max_num_emb=float('Inf'), tile=None):
```
Comment (kevinchern) [inline minorminer/utils/raster_embedding.py:172]: need to update documentation

File: minorminer/utils/raster_embedding.py [hunk 2]
```diff
@@ -202,18 +202,32 @@ def raster_embedding_search(S, T, timeout=10, raster_breadth=None,
...
+            tile = dnx.pegasus_graph(m=raster_breadth)
     elif T.graph.get('family') == 'zephyr':
         sublattice_mappings = dnx.zephyr_sublattice_mappings
         t = T.graph['tile']
-        tile = dnx.zephyr_graph(m=raster_breadth, t=t)
+        if tile is None:
+            tile = dnx.zephyr_graph(m=raster_breadth, t=t)
     else:
-        raise ValueError("source graphs must a graph constructed by "
+        raise ValueError("source graphs must a Zephyr graph constructed by "
```
Comment (kevinchern) [inline minorminer/utils/raster_embedding.py:218]: this should be for target graph `T`

**Review by kevinchern — APPROVED** [review]

File: minorminer/utils/raster_embedding.py (hunk 1 above)
Comment (jackraymond) [reply minorminer/utils/raster_embedding.py:172]: tile (nx.Graph(): default=None): By default the target subgraphs are displaced dwave_networkx chimera, pegasus or zephyr graphs (tiles) parameterized by raster_breadth. A customized tile can replace these default subgraphs. Providing a tile makes raster_breadth redundant.  
OR SIMILAR

File: minorminer/utils/raster_embedding.py [hunk 3]
```diff
@@ -199,21 +199,36 @@ def raster_embedding_search(S, T, timeout=10, raster_breadth=None,
...
                          "dwave_networkx as chimera, pegasus or zephyr type")
 
+    # TO DO: Add a function that finds the minimal feasible raster_breadth for
```
Comment (jackraymond) [inline minorminer/utils/raster_embedding.py:221]: You need to rebase to code_dev. These lines need to be deleted. In the most recent version of code_dev these lines are deleted and shouldn't be put back in.

Comment (jackraymond) [reply minorminer/utils/raster_embedding.py:221]: Or just delete 221 - 228. These lines are made redundant by 196-198

**Review by jackraymond — CHANGES_REQUESTED** [review]

**Pull request closed without merging**